careers/working-at-cfpb/index.html

```diff
@@ -0,0 +1,184 @@
+{% extends "layout-side-nav.html" %}
+{% import "_vars-careers.html" as vars with context %}
+{%- set active_nav_id = 'working-at-cfpb' -%}
+
+{% block title -%}
+    Working at the CFPB
+{%- endblock %}
+
+{% block desc -%}
+    Information on careers at the CFPB.
+{%- endblock %}
+
+{% block css -%}
+    {{ super() }}
+    {# TODO: Resolve slick font issue.
+             For the slick stylesheet to pick up its font and img reference the
+             fonts and images found at /src/vendor/slick-carousel/slick/ should
+             be included in same directory as slick.css.
+             However, including the slick fonts makes the dots on the
+             /the-bureau/ page at mobile sizes enormous.
+    #}
+    <link rel="stylesheet" href="/static/css/slick.css">
+{%- endblock %}
+
+{% block content_modifiers -%}
+    {{ super() }}
+    careers-working-at-cfpb
+{%- endblock %}
+
+{% block content_main_modifiers -%}
+    {{ super() }}
+    content__flush-bottom
+{%- endblock %}
+
+{% block hero -%}
+    <section class="content_hero hero">
+        <div class="hero_wrapper">
+        <div class="hero_img-container hero_img-container__small">
+            {# TODO: Replace placeholder div with image #}
+            <div class="hero_img"
+                 style="width:660px;height:300px;background-image:url({{ 'http://placehold.it/660x300.jpg' }});">
+            </div>
+        </div>
+        <div class="hero_img-container hero_img-container__large">
+            {# TODO: Replace placeholder div with image. #}
+            <div class="hero_img"
+                 style="width:1200px;height:300px;background-image:url({{ 'http://placehold.it/1200x300.jpg ' }});">
+            </div>
+        </div>
+    </section>
+{%- endblock %}
+
+{% block content_main %}
+
+    <section class="block
+                    block__flush-top
+                    block__sub">
+            <h1>Working at the CFPB</h1>
+            <p class="h3">
+                Our mission stems from our commitment to serving the consumer
+                and our country.
+                Our vision is a consumer finance marketplace that works for
+                American consumers, responsible providers,
+                and the economy as a whole.
+            </p>
+            <p class="short-desc">
+                This vision can only come from the people who are working
+                toward it.
+                Creating this kind of market takes a dedication to service,
+                leadership and innovation.
+            </p>
+    </section>
+
+    <div>
```

Comment:
I could of sworn I had them there for a reason, but I can't seem to find out why. Removed and the layout looks the same in https://github.com/cfpb/cfgov-refresh/commit/295f54d5581add948b58a701ca63ae0f6f8e2bce